lib/utils/sendInterviewTimes.ts

```diff
@@ -0,0 +1,60 @@
+import { SESClient } from "@aws-sdk/client-ses";
+import sendEmail from "../../utils/sendEmail";
+import {
+  emailApplicantInterviewType,
+  emailCommitteeInterviewType,
+  interviewType,
+} from "../types/types";
+const { applicantTestData, committeeTestData } = require("./tempEmailTestData");
+
+export const sendInterviewTimes = async () => {
+  const sesClient = new SESClient({ region: "eu-north-1" });
+
+  // Send email to each applicant
+  for (const applicant of applicantTestData) {
+    const typedApplicant: emailApplicantInterviewType = applicant;
+    const applicantEmail = [typedApplicant.applicantEmail];
+    const subject = `Hei, ${typedApplicant.period_name}. Her er dine intervjutider!`;
+    let body = `Intervjutider:\n\n`;
+
+    typedApplicant.committees.forEach((committee: interviewType) => {
+      body += `Komite: ${committee.name}\n`;
+      body += `Start: ${committee.interviewTimes.start}\n`;
+      body += `Slutt: ${committee.interviewTimes.end}\n\n`;
+    });
```

Comment:
Vi kommer vel ikke unna å få med rom i denne eposten, gjør vi?

Reply:
> Vi kommer vel ikke unna å få med rom i denne eposten, gjør vi?

Oi rom... Har vi egentlig tenkt på hvordan vi skal gjøre det med rom, og hvordan søkerne skal vite hvilket rom de har intervju i?

Reply:
> Vi kommer vel ikke unna å få med rom i denne eposten, gjør vi?

Burde jo egt sette det opp i frontenden at komiteene legger med rom, når de legger in tilgjengelige tider? Har algoritmen tenkt på rom @jorgengaldal ?

Reply:
Algoritmen trenger vel egentlig ikke å tenke på rom? Så lenge komiteer booker rom for tidene de har sagt så burde det vell være good? Tror alt vi trenger er mulighet for komiteene å legge inn navn på rom når de setter tider, slik at denne infoen kan sendes på mail til søkerne.